sources/packages/api/src/services/user/user.service.ts

```diff
@@ -14,4 +15,26 @@ export class UserService extends DataModelService<User> {
       userName,
     });
   }
+
+  async synchronizeUserInfo(userInfoBCServiceCard: UserInfo): Promise<User> {
+    let userToSync = await this.getUser(userInfoBCServiceCard.userName);
+
+    //Initializing a new user if one is not found
+    if (!userToSync) {
+      userToSync = new User();
```

Comment:
we should throw error here.

Reply:
We need to save the information so shouldn't be an error. Can discuss if needed